README.md

```diff
@@ -2,7 +2,17 @@
 
 Dashboard is a data-visualization platform for [GreptimeDB](https://github.com/greptimeteam/greptimedb) developed with Vue3, Vite, TypeScript and Arco Design.
 
-## Quick Start with Docker
+## Quick Start
+
+You are welcome to try out our dashboard directly with GreptimeDB's binary or with Docker.
+
+### 1. With GreptimeDB's binary
+
+- Download GreptimeDB's pre-built binary [here](https://docs.greptime.com/installation/download-pre-built).
```

Comment:
We can guide user to our new download page: https://greptime.com/downlod because there are multiple ways to download and all of them are available for dashboard